.gitignore

```diff
@@ -1,3 +1,2 @@
 node_modules
-blindsecp256k1-browser.js
 dist
```

Comment:
should the `dist` not be in the gitignore too?

Reply:
I actually thought that, but since it's being referenced in the `package.json` file seems that it works well this way. And I actually have the `dist` folder ignored in most of my projects, is the usual way to proceed since we don't know the dist code in git. But tbh I didn't know until today that (some) `.gitignored` files where also `.npmignored` 😕